app/views/lead_providers/guidance/reference.html.erb

```diff
@@ -112,4 +112,50 @@
   with a <code>401</code> status code.
 </p>
 
+<h3 id="custom-headers" class="govuk-heading-m">Custom API Headers for Sandbox testing</h3>
+
+<p class="govuk-body">
+  The sandbox environment supports the following custom header to enable testing:
+  <code>
+    X-With-Server-Date
+  </code>
+</p>
+
+<h4 id="x-with-server-date">
+  Usin the X-With-Server-Date header
+</h4>
+
+<p class="govuk-body">
+  Declaration submissions are made by providers must fall into specific date periods in line with contractual milestones. These are required to fall into specific date periods.
+</p>
+<p class="govuk-body">
+  To test the integration more thoroughly, a custom JSON header can be used when making declarations in the sandbox.
+</p>
+<p class="govuk-body">
+  This header, X-With-Server-Date, is set as a standard JSON header to simulate declaration submissions againstfor future milestone date periods.
+</p>
```

Comment:
The header doesn't really have much to do with JSON, it's just a request header, which should work regardless of the request format, right?